imap_processing/swapi/l1/swapi_l1.py

```diff
@@ -532,8 +526,9 @@ def process_swapi_science(sci_dataset: xr.Dataset, data_version: str) -> xr.Data
     )
 
     # L1 quality flags
+    # TODO: Should these be kept in raw format rather than derived into strings?
```

Comment:
keep it raw for now. I will ask SWAPI team if we want in derived value.

Reply:
OK, I reverted this change and set `use_derived_value=False` for SWAPI. See the latest commit for the update.

Reply:
thank you!